pulp_container/app/serializers.py

```diff
@@ -696,3 +707,17 @@ class Meta:
             "tag",
             "artifacts",
         )
+
+
+class ContainerRepositorySyncURLSerializer(RepositorySyncURLSerializer):
+    """
+    Serializer for Container Sync.
+    """
+
+    signed_only = serializers.BooleanField(
```

Comment:
Should this option affect only the current sync or should it remove content with no signatures from the existing repo version as well?
I added it as a one-time option to a sync operation. Should be a permanent one? on a repo maybe?

Reply:
i think it should affect only current sync. it user would want to remove content with no signature from the existing repo version he'd use mirror=True, wdyt?

Reply:
Sounds reasonable